.helmignore

```diff
@@ -2,3 +2,21 @@
 .terraform/
 bin/
 test/
+
+### Scoop additions ###
+
+.DS_Store
+.circleci/
+
+# Common backup files
+*.swp
+*.bak
+*.tmp
+*~
+*.md
+
+# Various IDEs
+.project
+.idea/
+*.tmproj
+.vscode/
```

Comment:
You can also ignore:

.gitignore
.helmignore
CHANGELOG.md
CONTRIBUTING.md
LICENSE.md
Makefile
README.md

Reply:
*.md files are excluded there, I'll add these ones in but I dont see a huge benefit to ignoring everything possible since the download and install is not visible to the user and the big ones are ignored already

Reply:
Oh, good point about *.md. Yeah, this isn't critical. But, it is nice to keep the chart clean. There's really no value in having stray files hanging out in it.